__tests__/comments.ts

```diff
@@ -978,6 +978,22 @@ describe('mutation commentOnPost', () => {
       expect(comment.flags).toEqual({ vordr: false });
     });
 
+    it('should set correct vordr flags on good user if vordr filter catches it', async () => {
```

Comment:
Maybe name them explicit for post/comment jest doesn't like same names so much and I actually thought for a second it was duplicated 😅

Reply:
Renamed tests